feat: OPTIC-1265: Improve error pages

mcanu opened the pull request:
<!--

This description MUST be filled out for a PR to receive a review. Its primary purposes are:

 - to enable your reviewer to review your code easily, and
 - to convince your reviewer that your code works as intended.

Some pointers to think about when filling out your PR description:
 - Reason for change: Description of problem and solution
 - Screenshots: All visible changes should include screenshots.
 - Rollout strategy: How will this code be rolled out? Feature flags / env var / other
 - Testing: Description of how this is being verified
 - Risks: Are there any known risks associated with this change, eg to security or performance?
 - Reviewer notes: Any info to help reviewers approve the PR
 - General notes: Any info to help onlookers understand the code, or callouts to significant portions.

You may use AI tools such as Copilot Actions to assist with writing your PR description (see https://docs.github.com/en/copilot/using-github-copilot/using-github-copilot-for-pull-requests/creating-a-pull-request-summary-with-github-copilot); however, an AI summary isn't enough by itself. You'll need to provide your reviewer with strong evidence that your code works as intended, which requires actually running the code and showing that it works.

-->

![image](https://github.com/user-attachments/assets/18b44195-9b32-48af-b44b-1344855f46a5)
![image](https://github.com/user-attachments/assets/601afbb7-886d-4cd4-9f46-2fa478a166ea)
![image](https://github.com/user-attachments/assets/642f5410-539f-45a4-a2fd-e9619396ee42)

mcanu committed:
Updated 404 page

mcanu committed:
Updated 403 and 500. Removed unused 401.html

mcanu committed:
Fix page title in 500.html

mcanu committed:
Improved buttons css. Removed custom 500 view

mcanu committed:
Fix linter issues

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13775325348)

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-optic-1265'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13775325348

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13775325348

yyassi-heartex commented on label_studio/core/static/css/error.css:
```diff
@@ -1,56 +1,35 @@
 .ls-global-error__heidi {
-  display: block;
-  margin: 32px auto;
+  width: 810.91px;
```
technically, its fine - just remember since its parts of a pixel browsers will likely round up and render it as 811px - others might act funny and floor the value. might be best to set it to 811px or check the figma, usually we're able to just copy the css definition right from there (not sure if the figma actually provided this value)

mcanu replied:
just got it from figma, I updated it to be 811px

ricardoantoniocm replied:
I've updated the designs to remove the decimals from the artwork. @mcanu you may grab the assets again.

mcanu committed:
Using integer pixels

mcanu committed:
Updated assets

mcanu commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13796121918)
> Successfully merged:  21 files changed, 288 insertions(+), 70 deletions(-)

mcanu committed:
Merge branch 'develop' into 'fb-optic-1265'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13796121918

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/13797750968)

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/13797750968
